### ISSUE TITLE
Control with the arrow keys

### ISSUE THREAD

**Issue by Drive4ik** [miaupaw/instant-eyedropper#7]
The app is great! The only thing missing is the pixel-by-pixel movement of the cursor. Sometimes the mouse is not very accurate and I would like more precise control of the keys (arrows) in color selection mode.
Thank you!

**Comment by Dhyfer1** [comment]
I agree with you. With screen color picker of ShareX I can use the arrow keys to move the crosshair one pixel at a time.

@miaupaw Can you update instant eyedropper to add this useful function?
